portal/templates/utilisation-report-service/record-correction/provide-utilisation-report-correction.njk

```diff
@@ -36,128 +36,131 @@
 
     <div class="govuk-grid-column-two-thirds-from-desktop">
 
-      {% if "FACILITY_ID_INCORRECT" in correctionRequestDetails.reasons %}
-        {{ govukInput({
-          label: {
-            text: "Enter the facility ID",
-            classes: "govuk-label--s"
-          },
-          hint: {
-            text: "The facility ID can be found on the deal summary page"
-          },
-          id: "facilityId",
-          name: "facilityId",
-          value: formValues.facilityId,
-          classes: "govuk-input--width-20",
-          attributes: {
-            "data-cy": "FACILITY_ID_INCORRECT-input"
-          }
-        }) }}
-      {% endif %}
-
-      {% if "REPORTED_CURRENCY_INCORRECT" in correctionRequestDetails.reasons %}
-        {{ govukRadios({
-          fieldset: {
-            legend: {
-              text: "Select the reported currency",
-              classes: "govuk-fieldset__legend--s"
+      <form method="post">
+
+        <input type="hidden" name="_csrf" value="{{ csrfToken }}"/>
+
+        {% if "FACILITY_ID_INCORRECT" in correctionRequestDetails.reasons %}
+          {{ govukInput({
+            label: {
+              text: "Enter the facility ID",
+              classes: "govuk-label--s"
+            },
+            hint: {
+              text: "The facility ID can be found on the deal summary page"
```

Comment:
data-cy for hint please and label

Reply:
same for the rest of this file

Reply:
```
fieldset: {
            legend: {
              html: "<legend class='govuk-fieldset__legend govuk-fieldset__legend--l'><h1 class='govuk-fieldset__heading' id='heading' data-cy='heading'>" + CONTENT_STRINGS.PAGE_TITLE + "</h1></legend>"
            }
          },
```

Not a blocker for now, can be done for e2e tests - can use html and put the html on the top of the page as a set so we do html: legendHtml